File: server/src/main/java/br/com/samayah/happyJobs/model/CV.java
package br.com.samayah.happyJobs.model;

import java.util.List;

import javax.persistence.CascadeType;
import javax.persistence.JoinColumn;
import javax.persistence.OneToMany;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.NonNull;


@Data
@NoArgsConstructor
public class CV {
	
	@NonNull
	private String titulo;
	
	@NonNull
	private String resumo;
		
	@OneToMany( cascade = CascadeType.ALL )
	@JoinColumn(name = "username")
	private List<Formacao> formacao;

	@OneToMany( cascade = CascadeType.ALL )
	@JoinColumn(name = "username")
	private List<ExperienciaProfissional> xpProf;
	
	//List<OutraExperiencia> outrasXpsLista;
	
	//List<String> certificacoesLista;
}

File: viewEmpresa/src/app/model/CV.ts
import { Formacao } from './Formacao';
import { XpProfissional } from './XpProfissional';

export class CV {

    resumo: string = '';
    titulo: string = '';

    formacao: Formacao[] = new Array<Formacao>();
    xpProf: XpProfissional[] = new Array<XpProfissional>();
        
    constructor(formacao?: Formacao[], xpProf?: XpProfissional[]){        
        this.formacao = formacao;
        this.xpProf = xpProf;
        
        console.log('this.formacao');
        console.log(this.formacao);
        console.log('this.xpprof');
        console.log(this.xpProf);
    }

}
